Makefile

```diff
@@ -3,15 +3,29 @@ help:
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | \
 	awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
-.PHONY: test
-test: build ## run tests
-	forge test --match-contract RiftTokenTest
-	forge test --match-contract CoreTest
-	forge test --match-contract UniswapVaultTest
-	forge test --match-contract MasterChefVaultTest
-	forge test --match-contract MasterChefV2VaultTest
-	forge test --match-contract NativeVaultTest
+check-%:
+	@ if [[ -z "${${*}}" ]]; then \
+        echo "Environment variable $* not set"; \
+        exit 1; \
+    fi
+
+.PHONY: bootstrap
+bootstrap: check-ALCHEMY_API_KEY install ## bootstrap project
+	@sed 's/{YOUR_API_KEY}/${ALCHEMY_API_KEY}/g' foundry.toml > foundry.temp
+	@mv foundry.temp foundry.toml
+
+.PHONY: install
+install: ## install dependencies
+	yarn && forge update
+
+.PHONY: clean
+clean: ## clean build artifacts
+	forge clean
 
 .PHONY: build
-build: ## clean and build contracts
-	forge clean && forge build
+build: clean ## build contracts
+	forge build
+
+.PHONY: test
+test: build ## run tests
```

Comment:
`forge test` automatically rebuilds any modified contracts so we can just remove this build step and save some time